examples/test_examples.py

```diff
@@ -849,6 +849,52 @@ def test_text_to_image_lora_checkpointing_checkpoints_total_limit_removes_multip
                 {"checkpoint-6", "checkpoint-8", "checkpoint-10"},
             )
 
+    def test_text_to_image_lora_with_text_encoder(self):
+        pretrained_model_name_or_path = "hf-internal-testing/tiny-stable-diffusion-pipe"
+
+        with tempfile.TemporaryDirectory() as tmpdir:
+            # Run training script with checkpointing
+            # max_train_steps == 7, checkpointing_steps == 2, checkpoints_total_limit == 2
+            # Should create checkpoints at steps 2, 4, 6
+            # with checkpoint at step 2 deleted
```

Comment:
I think we can safely ignore this no? We're not checking for checkpointing here.

Reply:
```suggestion
```